### PR TITLE
Bug 1214207 - Update notification_helper to l20n. r=gandalf

### PR DESCRIPTION
**Pull request by stasm**
https://bugzilla.mozilla.org/show_bug.cgi?id=1214207

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dbf4721f8cb3bfaae504522f6b19f4666227285c

File: shared/js/l10n.js [hunk 1]
```diff
@@ -2141,18 +2141,22 @@
...
-      }),
-    formatValues: (...keys) => {
-      const resp = keys.map(key => {
+    formatValue: function(id, args) {
+      return navigator.mozL10n.formatValue(id, args);
+    },
+    translateFragment: function (frag) {
+      return Promise.resolve(navigator.mozL10n.translateFragment(frag));
+    },
+    ready: function() {
```
Comment (zbraniecki) [inline shared/js/l10n.js]: can you fix this as you're on it? It should be a getter, not a function

Comment (stasm) [inline shared/js/l10n.js]: sure, good catch.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a8cd1810f48db17bbc24aca8e73ca13256d07af7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1391dfab13355d2ca5b08b681ddd675dbfa1b8a8

**Pull request merged**